srcpkgs/CalculiX-cgx/files/README_LICENSES

```diff
@@ -0,0 +1,5 @@
+cgx program is released under GNU GPL v2 or later (except for a few functions, see CGX_LICENSE)
```

Comment:
I don't think this file is necessary. having the licences installed should be enough

Reply:
I admit this file has limited interest. I wanted to make explicit the fact that libSN is released under GNU GPL v2 (or later), which I did not find that obvious given the multiple licenses of the package. But since the package license is GPL v2 + custom  and there is no license file for libSN, I guess it implicitely means that libSN is released under GPL v2.

Thus I will remove this file as you suggest.